logging_mock.go

```diff
@@ -0,0 +1,64 @@
+package appwrap
+
+import (
+	"github.com/stretchr/testify/mock"
+	"net/http"
+)
+
+type LoggingMock struct {
+	mock.Mock
+	Log Logging // all log calls are passed through to this logger (NullLogger may be used)
```

Comment:
Happy to move appengine usage to this, since it is _mostly_ shamelessly copied from appengine.